test/extended/testdata/samplepipeline-withenvs.yaml

```diff
@@ -0,0 +1,332 @@
+apiVersion: v1
```

Comment:
you should be able to strip a bunch of stuff out of this sample:

no need to have DB service/deployment
no need to have the app service/deployment (you just need the build)